scripts/validateSchemas.js

```diff
@@ -23,10 +23,14 @@ const ajvValidator = () => {
 };
 
 const schemaDefinitionJsonToExamplesGlob = {
```

Comment:
Minor - as it's no longer using a file glob here, it might be worth tweaking the var name? 

Also, this might get tricky to manage as it grows. I think it's okay for now but might be one to revisit?

Reply:
Removed the glob as part of the var name, and I agree with your second point 👍